gamestonk_terminal/portfolio/yfinance_model.py

```diff
@@ -0,0 +1,76 @@
+"""Yfinance model"""
+__docformat__ = "numpy"
+
+from typing import List
+
+import yfinance as yf
+import pandas as pd
+
+
+def get_stocks(tickers: List[str]) -> pd.DataFrame:
+    """Gets historic data for list of tickers
+
+    Parameters
+    ----------
+    tickers : List[str]
+        Tickers to get data for
+
+    Returns
+    ----------
+    data : pd.DataFrame
+        Historic daily prices for stocks
+    """
+
+    df = yf.download(tickers=tickers, period="5y", interval="1d", progress=False)
```

Comment:
I think you should have a variable start date.  Instead of doing the period of 5yr, you could have a start variable, then when you call this in portfolio stuff, just default the start to df.index[0]-timedelta(days=255) so you can calculate beta.